app/controllers/admin/tenants_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Admin::TenantsController < Admin::BaseController
+
+  respond_to :html
```

Comment:
Considering all responses are HTML (at least for this controller), do we really need this line?